readthedocs/proxito/views/hosting.py

```diff
@@ -274,12 +290,26 @@ def get(
         best JSON structure for that particular version.
         """
         if addons_version.major == 1:
-            return self._v1(project, version, build, filename, url, user)
+            return self._v1(project, version, build, filename, url, request)
 
         if addons_version.major == 2:
-            return self._v2(project, version, build, filename, url, user)
+            return self._v2(project, version, build, filename, url, request)
+
+    def _get_active_versions(self, request, project):
+        """
+        Get all active for a project that the user has access to.
+
+        This includes only active and built versions that are not hidden.
+        """
```

Comment:
```suggestion
    def _get_versions(self, request, project):
        """
        Return versions matching the following conditions:
        
         - the user has access to
         - they are built
         - they are active
         - they are not hidden
        """
```

I'm making this method more explicit and clearer to avoid confusions, since they are not just "active" versions.